style.css

```diff
@@ -1,112 +1,96 @@
-
-*{
-    text-decoration: none;
-    list-style: none;
-    padding: 0;
-    margin: 0;
+* {
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/133863470-cba1c229-c867-46f2-b1a4-c9327185c56f.png)

- [ ] Kindly apply the necessary styling to make your webpage look exactly like the original design